src/Cthulhu.jl

```diff
@@ -511,7 +511,7 @@ function _descend(term::AbstractTerminal, interp::AbstractInterpreter, curs::Abs
                     ioctx = IOContext(iostream,
                         :color => true,
                         :displaysize => displaysize(iostream), # displaysize doesn't propagate otherwise
-                        :SOURCE_SLOTNAMES => Base.sourceinfo_slotnames(codeinf),
+                        :SOURCE_SLOTNAMES => codeinf === nothing ? false : Base.sourceinfo_slotnames(codeinf),
```

Comment:
Is this change necessary? `codeinf` here should always be `CodeInfo` object.

Reply:
We did need it, yes, I'll send you a MWE.